src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5278,7 +5278,7 @@ class C
             int exitCode = csc.Run(outWriter);
             Assert.Equal(0, exitCode);
             Assert.Equal(@"
-Microsoft (R) Visual C# Compiler version A.B.C.D
+Microsoft (R) Visual C# Compiler version A.B.C.D (<developer build>)
```

Comment:
This will actually fail when the job is commited, as (unlike PRs) commit jobs are built with `OfficialBuild=true`.